setup.py

```diff
@@ -153,7 +160,7 @@
     license      = 'MIT',
     platforms    = [ 'any' ],
     packages     = packages,
-    requires     = [ 'dnspython', 'pyasn1', 'pyasn1_modules' ],
+    requires     = python_requirements,
```

Comment:
The keyword should be `install_requires` instead of `requires` according to the docs http://python-packaging.readthedocs.org/en/latest/dependencies.html#specifying-dependencies .